spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -47,9 +47,9 @@ def assert_specific_datasource(nativeid)
   end
 
   def assert_specific_domain
-    domain = @ems_hawkular.middleware_domains.find_by_name('master')
+    domain = @ems_hawkular.middleware_domains.find_by_feed('master.Unnamed%20Domain')
```

Comment:
Rails used to have dynamically named methods prefixed with `find_by_` (so, if you had a model with a `hair_color` attribute, you could do `Person.find_by_hair_color(<color>)`. Use of these methods has been deprecated or removed (as `find_by(color: <color>)` is a much less magical and better way)

I said `.find_by(:feed => 'master.Unnamed%20Domain')` instead of @mtho11's solution because it does the exact same behavior as what's there and is more clear. (you would have to add a `.first` if adding a `where` call instead)

For more info: http://api.rubyonrails.org/v5.0.0.1/classes/ActiveRecord/FinderMethods.html#method-i-find_by